hive/replays/recurrent_replay.py

```diff
@@ -0,0 +1,262 @@
+import os
+import pickle
+
+import numpy as np
+from hive.replays.circular_replay import CircularReplayBuffer
+
+
+class RecurrentReplayBuffer(CircularReplayBuffer):
+    """
+    First implementation of recurrent buffer without storing hidden states
+    """
+
+    def __init__(
+        self,
+        capacity: int = 10000,
+        max_seq_len: int = 1,
+        n_step: int = 1,
+        gamma: float = 0.99,
+        observation_shape=(),
+        observation_dtype=np.uint8,
+        action_shape=(),
+        action_dtype=np.int8,
+        reward_shape=(),
+        reward_dtype=np.float32,
+        extra_storage_types=None,
+        num_players_sharing_buffer: int = None,
```

Comment:
Why is the spacing inconsistent?

Reply:
If we specify the datatype black will add whitespaces around the equal sign, otherwise it will remove those whitespaces.